docs/user/supported_devices.rst

```diff
@@ -111,6 +111,7 @@ ath79-generic
 
 * Sophos
 
+  - AP15
```

Comment:
```suggestion
```

broken devices aren't listed in the docs

Reply:
Ah, my bad!